docker-compose.yml

```diff
@@ -377,29 +375,32 @@ services:
     # L3VXLAN Symmetric IRB:
     #   Ping to Bleaf internet connectivity IPs from Host1 via green and yellow VRFs from Leaf1
     command: |
-      sh -x -c 'sleep 30 && \
-                ip link add link eth0 name eth0.10 type vlan id 10 && \
+      sh -x -c 'ip link add link eth0 name eth0.10 type vlan id 10 && \
                 ip link set eth0 up && ip addr flush dev eth0 && \
                 ip link set eth0.10 up && \
                 ip addr add 10.10.10.10/24 dev eth0.10 && \
+                sleep 5 && \
                 ping -c 3 10.10.10.11 && \
-                ip link add link eth1 name eth1.20 type vlan id 20 && \
-                ip link set eth1 up && ip addr flush dev eth1 && \
-                ip link set eth1.20 up && \
-                ip addr add 20.20.20.20/24 dev eth1.20 && \
+                ip link add link eth0 name eth0.20 type vlan id 20 && \
+                ip link set eth0 up && ip addr flush dev eth0 && \
```

Comment:
You have ran this allready in the previous step. Why do you need to run it again ?

Reply:
We dont need this, copy & paste mistake.

Reply:
Cleaned it up. Thanks for the catch.